src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Orchestrators/PledgeOrchestratorTests.cs

```diff
@@ -452,6 +452,31 @@ public async Task GetApplications_Returns_Valid_ViewModel()
 
         }
 
+        [TestCase(true, true)]
+        [TestCase(false, false)]
+        public async Task GetApplications_Returns_Valid_ViewModel_With_UserCanClosePledge(bool OwnerOrTransactorStatus, bool expectWhetherUserCanClosePledges)
```

Comment:
Pedantic, but your parameter name should be lower case

Reply:
yes sure, the parameter name is now in lower case. I have updated it.